GamingTheSyllabus/Assets/Scripts/Firebase/DatabaseManager.cs

```diff
@@ -19,8 +20,10 @@ public class DatabaseManager : Singleton<DatabaseManager>
 
     private static Dictionary<Loadable, bool> loadingStatus = new Dictionary<Loadable, bool>();
 
-    private static FirebaseStorage databaseStorage;
-    private static StorageReference rootDirectory;
+    private static FirebaseStorage firebaseStorage;
+    private static StorageReference storageRootDirectory;
+    private static FirebaseDatabase firebaseDatabase;
+    private static DatabaseReference databaseRootDirectory;
```

Comment:
Bless you for renaming the storage root directory var to not be ambiguous with the new one